northstar/src/runtime/process/mod.rs

```diff
@@ -53,17 +52,29 @@ pub(super) struct Launcher {
     config: Config,
 }
 
+/// Container application information.
+struct ContainerApplication {
+    path: CString,
+    argv: Vec<CString>,
+    env: Vec<CString>,
+}
+
 pub(super) struct Process {
-    pid: Pid,
-    checkpoint: Option<Checkpoint>,
+    init_tx: channel::AsyncSender<InitApi>,
+    init_rx: Option<channel::AsyncReceiver<InitMessage>>,
+    console_stop: Option<CancellationToken>,
+    container: Container,
+    container_app: ContainerApplication,
```

Comment:
I'm not happy with `container_app`. A process in a os has also attributes like `cmd`, `args, `env`. Maybe just flatten it here?

Reply:
Yep, i don't like it much either, i'll inline it.